website/docs/components/form/segmented-group/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,116 @@
+## Usage
+
+A Segmented Group is an agnostic container that wraps two or more `Segments`, making the relevant use cases broad.
+
+!!! Info
+
+How these components are published and assembled is fundamentally different between the Figma and Ember components.
+
+- In Figma, we publish multiple primitive `Segment` components (`Segmented Button`, `Segmented Toggle`, and `Segmented Input`) that are intended to be assembled in an auto layout container. A pre-assembled `Base` component is also made available for more simple instances.
+- The Ember component is a container that enforces consistent styling and grouping of nested Helios form components. It does not manage any logic or functionality out of the box, these are left up to the consumer to implement based on the nested components.
+!!!
+
+### When to use
+
+- When filtering complex data, searching within a set of filters, and in creating complex filter patterns.
+- When collecting information in a way that benefits from a "connected" visual representation, e.g., a key/value pair.
+- When an action performs a generative function rather than a submit function, e.g., generating API credentials.
+- As a complex data input within a form.
+
+### When not to use
+
+- To submit a form or set of inputs. A Segmented Group can be used within a form to collect complex data, but an action within the group should not submit that data via a form method. Use the guidelines in [Form patterns](/patterns/form-patterns) instead.
+- To connect multiple fields that each require their own label.
+- When adding a prefix or suffix to an input field. This is not currently supported by Helios form components, instead consider using a simple text element preceding or following the form element.
+
+## Segment components
+
+A Segmented Group can consist of multiple primitive Segments that vary in their properties and intended input value. We recommend limiting usage in Figma to the defined `Segmented Button`, `Segmented Toggle` and `Segmented Input` primitives to ensure consistent styling and grouping. The Ember component supports a generic element in either the leading or trailing position if a custom solution is required.
```

Comment:
See comment below about "Toggle" vs "Dropdown" (and check in other parts of the documentation too)

```suggestion
A Segmented Group can consist of multiple primitive Segments that vary in their properties and intended input value. We recommend limiting usage in Figma to the defined `Segmented Button`, `Segmented Dropdown` and `Segmented Input` primitives to ensure consistent styling and grouping. The Ember component supports a generic element in either the leading or trailing position if a custom solution is required.
```

Reply:
@alex-ju the statement "The Ember component supports a generic element in either the leading or trailing position if a custom solution is required" I think is not accurate: since it's a yielded component, it can be in any position.

Reply:
Indeed, it can be anywhere, leading, trailing, or anywhere in the middle.